src/main/java/thePackmaster/actions/evenoddpack/BattleCryAction.java

```diff
@@ -0,0 +1,23 @@
+package thePackmaster.actions.evenoddpack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.StrengthPower;
+import thePackmaster.util.Wiz;
+
+public class BattleCryAction extends AbstractGameAction {
+    
+    public BattleCryAction(int amount) {
+        this.amount = amount;
+    }
+    
+    public void update() {
```

Comment:
Please make this an anonymous action, they help centralizing the logic and keep the project easier to oversee for me